yarn-project/noir-protocol-circuits/src/crates/types/src/tests/fixtures/read_requests.nr

```diff
@@ -8,42 +8,32 @@ use crate::constants::{
         MAX_READ_REQUESTS_PER_CALL,
 };
 
-pub fn generate_read_requests(how_many: Field) -> (
-    BoundedVec<SideEffect, MAX_READ_REQUESTS_PER_CALL>,
-    BoundedVec<ReadRequestMembershipWitness, MAX_READ_REQUESTS_PER_CALL>
-) {
+pub fn generate_read_requests(how_many: Field) -> (BoundedVec<SideEffect, MAX_READ_REQUESTS_PER_CALL>, BoundedVec<ReadRequestMembershipWitness, MAX_READ_REQUESTS_PER_CALL>) {
     generate_read_requests_with_config(how_many, false, [0; MAX_READ_REQUESTS_PER_CALL])
 }
 
-pub fn generate_transient_read_requests(how_many: Field) -> (
-    BoundedVec<SideEffect, MAX_READ_REQUESTS_PER_CALL>,
-    BoundedVec<ReadRequestMembershipWitness, MAX_READ_REQUESTS_PER_CALL>
-) {
+pub fn generate_transient_read_requests(how_many: Field) -> (BoundedVec<SideEffect, MAX_READ_REQUESTS_PER_CALL>, BoundedVec<ReadRequestMembershipWitness, MAX_READ_REQUESTS_PER_CALL>) {
```

Comment:
The formatting here is also strange, if the arguments are long they are split, but if the return values are they are still put on the same line.